data_model/master/clusters/JointFabricPKICluster.xml

```diff
@@ -68,29 +68,38 @@ Davis, CA 95616, USA
       <item value="0" name="ref_IcacsrRequestStatusOK" summary="No error">
         <mandatoryConform/>
       </item>
-      <item value="1" name="ref_InvalidIcacsr" summary="The ICACSR in the request was invalid">
+      <item value="1" name="ref_InvalidIcaCsrFormat" summary="The ICACSR in the request is not compliant to &lt;&lt;ref_Rfc2986, PKCS #10&gt;&gt; rules">
```

Comment:
This seems like a scraper bug: rather than listing the name from the table, we list the name of the cross reference.

Reply:
Though, TBF, the formatting in JFPC is very much non-standard: 

```
| Value   | Name                                                           | Summary                                                                                       | Conformance
| 0      s| [[ref_IcacsrRequestStatusOK]]OK                                | No error                                                                           
```

Reply:
yes, known issue the scraper. The plan for refs is to move to alchemy now that it's open sourced. However, I opted not the make the switch over until after we release the final TH package for 1.4. I don't want to add additional destabilization in here. The items here are not used in any of the automated tests.